app/Http/Resources/Group.php

```diff
@@ -65,6 +65,16 @@ class Group extends JsonResource
      *     )
      */
 
+    /**
+     *     @OA\Property(
+     *          property="phone",
+     *          title="phone",
+     *          description="An optional phone number to contact the group.",
+     *          format="string",
+     *          example="https://therestartproject.org"
```

Comment:
The example is wrong.

Reply:
Indeed it is.  Fixed (in develop, post merge).